tails_files/install.sh

```diff
@@ -21,6 +21,13 @@ DESKTOP=$HOMEDIR/Desktop
 ANSIBLE=$PERSISTENT/securedrop/install_files/ansible-base
 SSH_ALIASES=false
 
+# check for Tails 2.x
+source /etc/os-release
```

Comment:
Good to know it's a small file, and therefore the environment changes are minimal. (That's true today, though may not be true into the future!) Expected the answer to be that grepping with `set -e` can be complicated, since grep by default exits non-zero if there wasn't a match. Either way, an explanation of the decision should absolutely be added in a comment.